commands/solenoid/SolenoidSet.java

```diff
@@ -23,34 +26,21 @@
 	 * @param booleanSuppliers conditions that if true, prevents solenoidSubsystem
 	 *                         from setting
 	 */
-	public SolenoidSet(String name, SolenoidSubsystem system, SolenoidState state,
+	public SolenoidSet(SolenoidSubsystem system, SolenoidState state,
 			BooleanSupplier... booleanSuppliers) {
-		super(name, system);
 		this.system = system;
 		this.state = state;
 		this.booleanSuppliers = booleanSuppliers;
-	}
 
-	/**
-	 * Sets the state of a SolenoidSubsystem
-	 * 
-	 * @param system SolenoidSubsystem to set
-	 * @param state  state to set system
-	 */
-	public SolenoidSet(SolenoidSubsystem system, SolenoidState state, BooleanSupplier... booleanSuppliers) {
-		this("SolenoidSet", system, state, booleanSuppliers);
+		requirements = Set.of((Subsystem) system);
```

Comment:
Don't add this property. Requirements should be computed solely in `getRequirements`

Reply:
see #245.